src/error/LogHeroErrors.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace LogHero\Client;
+
+
+class LogHeroErrors {
+    private $errorFilePrefix;
+
+    public function __construct($errorFilePrefix) {
+        $this->errorFilePrefix = $errorFilePrefix;
+    }
+
+    public function setErrorFilenamePrefix($errorFilePrefix) {
+        $this->errorFilePrefix = $errorFilePrefix;
+    }
+
+    public function writeError($errorTypeId, $fullError) {
+        $errorFilename = $this->getErrorFilename($errorTypeId);
+        file_put_contents($errorFilename, $fullError);
```

Comment:
It's wanted that the file gets overwritten each time you write an error with the same type id?